ops/reth-cross-test/phases/fault_phase.py

```
@@ -0,0 +1,202 @@
+            self.teardown()
+
+            # Scenario E: Wipe reth sequencer data and re-sync
+            self.setup()
+            self.wait_ready()
+            self.wait_blocks(20)
+            scenarios["wipe_reth_sequencer"] = self._wipe_reth_sequencer()
+            self.teardown()
+
+            self._results = {"scenarios": scenarios}
+            return self.report()
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)